### ISSUE TITLE
Search app fails silently on bad facets query parameter

### ISSUE THREAD

**Issue by karlcz** [informatics-isi-edu/chaise#450]
Going to a search URL that has the facets query parameter with url-escaped parentheses fails with a blank page rather than displaying error information and/or recovering by discarding the bad parameter and loading default facet view.

**Comment by svoinea** [comment]
Fixed through commit 7ad26bc4b2363341d79e17a2e152ac099833c593
In case of such an error, an error page is rendered containing the reason of it.
Example of such a message for the URL `https://synapse-dev.isrd.isi.edu/chaise/search/#1/Zebrafish:Subject?facets=%28Subject:Issuer::eq::Dsy%29&layout=table&page=1`:
```
ERROR:
    malformed URI sequence
    facets=%28Subject:Issuer::eq::Dsy%29
```
@karlcz : I think it's better to display an error message and let the user correct it, than to pop up an alert window with that message and loading the default facet view.

**Comment by karlcz** [comment]
ok
